tests/Types/JsonTypeTest.php

```diff
@@ -1,50 +0,0 @@
-<?php
-
-use Doctrine\DBAL\Platforms\AbstractPlatform;
-use Doctrine\DBAL\Types\Type;
-use LaravelDoctrine\ORM\Types\Json;
-use Mockery as m;
-use Mockery\Mock;
-
-class JsonTypeTest extends PHPUnit_Framework_TestCase
```

Comment:
I think, that currently you should leave this test, because we need to be sure that we haven't broke anything

Reply:
As mentioned, this test have _no effect_. It just tests the JsonType from Doctrine.. I readded it anyway.